app/controllers/formbuilder/new.js

```diff
@@ -0,0 +1,166 @@
+import Controller from '@ember/controller';
+
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { service } from '@ember/service';
+import { restartableTask, timeout } from 'ember-concurrency';
+import {
+  DESCRIPTION_NOT_USED_PLACEHOLDER,
+  NAME_INPUT_CHAR_LIMIT,
+} from '../../utils/constants';
+import { getLocalFileContentAsText } from '../../utils/get-local-file-content';
+import { ForkingStore } from '@lblod/ember-submission-form-fields';
+import { FORM, RDF } from '@lblod/submission-form-helpers';
+import { PREVIEW_SOURCE_NODE } from './edit';
+
+export default class FormbuilderNewController extends Controller {
+  @service store;
+  @service router;
+  @service toaster;
+  @service intl;
+
+  @tracked name = '';
```

Comment:
because this is now in a controller, the name is not automatically deleted when closed

Reply:
This also made it possible to add 3 forms with the same name. But I think just fixing the name thingy will fix the duplicate names

Reply:
Ah yes because the input field is not focused it didn't check for duplicate names. Resetted the name to be null at start.